FreeRTOS/FreeRTOS
# Build: Fix Demo/RX100-RSK_Renesas_e2studio RTOS demo project build error

Description
-----------
This PR fixes Demo/RX100-RSK_Renesas_e2studio RTOS demo project build error.
There are no changes in the runtime functionality.

Test Steps
-----------
I did the following steps with e2 studio v1.1.1.10 and I used CC-RX V1.2.1. (According to .info file, the project was originally built with e2 studio v1.1.1.7 using CC-RX V1.2.1 (moreover, and very old source code of FreeRTOS perhaps 7 ~ 8 years ago). Unfortunately the project file format is too old to be updated by the e2 studio v7.8.0. Therefore remaking the project file is to be done in the future.)

(1) Run the following bat file in the Demo/RX100-RSK_Renesas_e2studio folder
CreateProjectDirectoryStructure.bat
(2) Comment out the following #error in the Demo/RX100-RSK_Renesas_e2studio/RTOSDemo/FreeRTOSConfig.h
#error Ensure CreateProjectDirectoryStructure.bat has been executed before building.  See comment immediately above.
(3) Import the project
(4) Clean all projects and start a build

But I don't have the evaluation board therefore I can't check the program which are built.
If this isn't acceptable for you, please let me know.

Related Issue
-----------
Update Renesas GCC compiler ports #135
https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/135#issuecomment-680172086
https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/135#issuecomment-679215454

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
